src/main.py

```diff
@@ -20,6 +20,7 @@ def __init__(self) -> None:
         self.shouldExit = False
         self.isCalibrated = False
         self.settings = {}
+        self.cameraFrameSize = (848, 480)  # 480p
```

Comment:
If you resize the image, you should probabally keep the same aspect ratio as the original camera image, otherwise it looks distorted and stretched.